### ISSUE TITLE
:LualineNotices warns for depreciation of nvim_lsp in favour of nvim_diagnostic

### ISSUE THREAD

**Issue by evoludigit** [lvim-tech/lvim#28]
Hello,

I am using lvim and I must say I love all the default configs so congratulations for this great work.

I get a depreciation message  while launching lvim:
`lualine: There are some issues with your config. Run :LualineNotices for details`

The command :LualineNotices then outputs:

### diagnostics.source
Diagnostics source `nvim_lsp` has been deprecated in favour of `nvim_diagnostic`.
nvim_diagnostic shows diagnostics from neovim's diagnostics api while nvim_lsp used to only show lsp diagnostics.

```
You've something like this your config.

lua
  {'diagnostics', sources = {'nvim_lsp'}}

It needs to be updated to:
lua
  {'diagnostics', sources = {'nvim_diagnostic'}}
```

I tried to fix the message by spotting the places wtih nvim_lsp and replacing it with nvim_diagnostic, but that did not work:

```
[me@linux] .config/nvim/ $ grep -rnw . -e nvim_lsp
./lua/modules/global/configs/completion/init.lua:67:                        nvim_lsp = "[LSP]", 
./lua/modules/global/configs/completion/init.lua:77:                    name = "nvim_lsp" 
./lua/modules/global/configs/ui/init.lua:547:            "nvim_lsp" 
```

being a noob in Lua, I have not managed to investigate the problem further.

thanks again for your very sleek config 👏

**Comment by blasterpistol** [comment]
It also breaks a fresh installation 
```
 ✗ Failed to install michaelb/sniprun
   Using lsp_document_diagnostics for Trouble is deprecated. Please use document_diagnostics instead.
```

**Comment by lvim-tech** [comment]
Fixed in latest commit - v1.3.0
